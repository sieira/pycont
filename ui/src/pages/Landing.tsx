import React from 'react';

import logo from '../logo.svg';
import './Landing.css';


const Landing: React.FC = () => (
  <div className="App">
    <header className="App-header">
      <img src={logo} className="App-logo" alt="logo" />
      <p>
        <span role="img" aria-label="beer">🍺 Edit</span>
        and save to reload.
        <br />
        <a href="/login">Let me in</a>
      </p>
    </header>
  </div>
);

export default Landing;
