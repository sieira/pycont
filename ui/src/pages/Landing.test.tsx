import React from 'react';
import { create } from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import thunkMiddleware from 'redux-thunk-recursion-detect';

import Landing from './Landing';

it('renders without crashing', () => {
  const div = document.createElement('div');
  const landingPage = create(<Landing />);
  expect(landingPage.toJSON()).toMatchSnapshot();
});
