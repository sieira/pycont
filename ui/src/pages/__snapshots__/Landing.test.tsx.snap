// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should render without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      <span
        aria-label="beer"
        role="img"
      >
        üç∫ Edit
      </span>
      and save to reload.
      <br />
      <a
        href="/login"
      >
        Let me in
      </a>
    </p>
  </header>
</div>
`;
